server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMailboxes.feature

```diff
@@ -242,3 +242,10 @@ Feature: GetMailboxes method
     Then the mailbox "shared2" has 0 messages
     And the mailbox "shared2" has 0 unseen messages
 
+  Scenario: User can share sub-mailbox without sharing its parent
+    Given "alice@domain.tld" has a mailbox "mailbox1"
+    And "alice@domain.tld" has a mailbox "mailbox1.shared"
+    And "alice@domain.tld" shares her mailbox "mailbox1.shared" with "bob@domain.tld" with "aeirwt" rights
+    When "bob@domain.tld" ask for mailboxes
+    Then the mailbox "shared" should be present
+    And the mailbox "mailbox1" should not be present
```

Comment:
If we have lookup right, which is supposed to be propagated, we are supposed to be able to see mailbox1

That can also be cool to mention the level of rights we get on mailbox1